packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx

```diff
@@ -1569,6 +1571,14 @@ export const ComponentShowcase = () => {
           })}
         </div>
       </div>
+      <div className='typeahead-select-list'>
+        <CWText type='h3'> Typeahead Dropdown</CWText>
+        <CWTypeaheadSelectList
+          options={optionList}
+          defaultValue={optionList[0]}
+          placeholder='Select chain'
```

Comment:
prettier should transform single quotes to double

Reply:
Updated!